core/chaincode/shim/chaincode.go

```diff
@@ -148,40 +147,20 @@ func getPeerAddress() string {
 	}
 
 	if peerAddress = viper.GetString("peer.address"); peerAddress == "" {
+		os.Exit(-1)
```

Comment:
Sure, will add error log.
How about:
`chaincodeLogger.Fatalf("Peer address not configured")`
instead
`os.Exit(-1)`